stable/jupyter/templates/jupyter-deployment.yaml

```diff
@@ -149,6 +149,11 @@ spec:
           - --namespace={{ .Release.Namespace }}
           - --service-name={{ .Release.Name }}
           - --instance-name={{ template "jupyter.fullname" . }}
+{{- if .Values.proxy.metricNames }}
+{{- range $val := .Values.proxy.metricNames }}
+          - --metric-names={{ $val }}
```

Comment:
Makes more sense to me that this would be `- --metric-name={{ $val }}` per metric. See: https://github.com/v3io/sidecar-proxy/pull/16#pullrequestreview-304071705